src/comparator/wrappers.py

```diff
@@ -154,23 +229,62 @@ def fields(self) -> Dict[int, Field]:
         Returns:
             Dict[int, Field]: Field is identified by number.
         """
+        # DescriptorProto.field has field number 2.
+        # So we append (2, field_index) to the path.
         return {
-            field.number: Field(field, self.file_resources, self.resource)
-            for field in self.message_pb.field
+            field.number: Field(
+                field,
+                self.proto_file_name,
+                self.source_code_locations,
+                self.path
```

Comment:
nit: This formatting looks a little ugly - can it just be `self.path + (2, i),` on a single line?

Reply:
This is auto-formatted by `black` module and unfortunately, if it is in the same line `python -m black` check would fail.  But we can ignore those specific lines by adding `# fmt: off` and `# fmt: on` to make sure it looks reasonable :) 
Same below, added annotation to force the formatter to ignore the blocks. Thanks.